js/wallet.js

```diff
@@ -40,7 +41,11 @@ class JS extends Wallet {
       request: request
     })
 
+    this.stateFile = require('../lib/data/ledger-state.json')
```

Comment:
Where do we use this one?

Reply:
This was just added to expose the base `ledger-state` file added in this PR. It isn't necessary at the moment though, will remove from this PR

Reply:
thx